Source/Infrastructure/Kafka/KafkaModule.cs

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) 2017 International Federation of Red Cross. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+using System;
+using Autofac;
+
+namespace Kafka
+{
+    public class KafkaModule : Autofac.Module
+    {
+        const string KAFKA_CONNECTIONSTRING = "KAFKA_CONNECTIONSTRING";
+
+        protected override void Load(ContainerBuilder builder)
+        {
+            var environmentVariables = Environment.GetEnvironmentVariables();
+            var kafkaConnectionString = "52.178.92.69:9092";
```

Comment:
Even though this is the fallback used if the connectionsting is not passed in as an environment variable, should the default be hardcoded?

Reply:
Agree, we should be using the .NET configuration where we can allow AppSettings and environment variables overrides - putting in an issue for it; #483